src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -0,0 +1,119 @@
+@snippet methodSampleDoc(docConfig)
+    @let ApiName = docConfig.getApiName, \
+            apiName = context.upperCamelToLowerCamel(ApiName), \
+            methodName = docConfig.getMethodName(), \
+            returnType = docConfig.getReturnType(), \
+            initCodeSpec = docConfig.getInitCode()
+        try {
+            ${@apiName} = new {@ApiName}();
+            {@initCode(initCodeSpec)}
+            @if docConfig.isPagedVariant()
+                for ({@methodCallSampleCode(methodName, apiName, initCodeSpec)} as element) {
```

Comment:
Is this possible? Because PHP does not have separate types, the "standard" way to tell if an array is associative is a runtime inspection... http://stackoverflow.com/questions/173400/how-to-check-if-php-array-is-associative-or-sequential
Maybe we should do the associative case by default?